app/file-upload/server/config/Webdav.js

```diff
@@ -5,7 +5,16 @@ import { settings } from '../../../settings';
 import '../../ufs/Webdav/server.js';
 
 const get = function(file, req, res) {
-	this.store.getReadStream(file._id, file).pipe(res);
+	this.store.getReadStream(file._id, file)
+		.on('error', () => {
+			console.error('An error ocurred when fetching the file');
+			res.writeHead(400);
```

Comment:
What can be handled better by the FE? A 500 with a message? The current 400 + log? A 404?

Reply:
I'd say we should reply either 503 or 500..

Reply:
A 503 should work  :)